docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/string-tensor-pack-15.rst

```diff
@@ -0,0 +1,110 @@
+.. {#openvino_docs_ops_type_StringTensorPack_15}
+
+StringTensorPack
+===================
+
+
+.. meta::
+  :description: Learn about StringTensorPack-15 - data movement operation which packs a concatenated batch of strings into a batched string tensor.
+
+**Versioned name**: *StringTensorPack-15*
+
+**Category**: *Data movement*
+
+**Short description**: *StringTensorPack* operation uses outputs generated by *StringTensorUnpack* to transform a concatenated batch of strings into 
+a string tensor.
+
+**Detailed description**
+
+Consider inputs:
+
+* *word_begins* = [0, 5]
+* *word_ends* = [5, 13]
+* *output_symbols* = "IntelOpenVINO"
+
+*StringTensorPack* uses indices from ``word_begins`` and ``word_ends`` to transform concatenated string ``output_symbols`` into ``output``, 
+a batched string tensor. The ``output.shape`` (also called ``batch_size``) is equal to ``word_beings.shape`` and ``word_ends.shape``, 
+and in this case holds values ``["Intel", "OpenVINO"]``.
+
+**Inputs**
+
+* **1**: *word_begins*:
```

Comment:
I think the `word` should be replaced by `string` (`string_begins`, `string_ends`) or dropped entirely (just `begins` and `ends`).

Reply:
Done.